### PR TITLE
base skeleton component setup

### PR DESCRIPTION
**Pull request by benschac**
mostly just looked at how `linear-gradient` was setup for this PR.

Goal is just to get all of the initial package setup done with.

Following PRs will implement functionality.

For this feature, I plan on merging the entire stack once the feature is complete. If you LGTM, I wouldn't merge until we have a working skeleton component.

### CODE REVIEW

**Comment by benschac** [comment]
Current dependencies on/for this PR:
* master
  * **PR #1867** <a href="https://app.graphite.dev/github/pr/tamagui/tamagui/1867?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/tamagui/tamagui/1867?utm_source=stack-comment).

File: packages/tamagui/src/index.ts [hunk 1]
```diff
@@ -28,6 +28,8 @@ export * from '@tamagui/select'
 export * from '@tamagui/separator'
 export * from '@tamagui/shapes'
 export * from '@tamagui/sheet'
+// Will uncomment when we have a skeleton component
```
Comment (stephenlaughton) [inline packages/tamagui/src/index.ts:31]: what does everyone think about standardizing on `// TODO: todo item.....` format for any todo comments for easy searchability? @natew @benschac

**Comment by benschac** [comment]
Yah, sound good

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by natew** [comment]
closing these unfinished ones you can re-open as a big one

**Pull request closed without merging**